lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
+    description: "enforces getter/setter pairs in objects",
+    ruleType: "Best Practices",
+    recommended: false,
+    default: [{getWithoutSet: false, setWithoutGet: true}]
```

Comment:
That's what I meant, if you're going to force people to specify default options, then we need to handle passing those options from the core.